apis/profiles.py

```diff
@@ -50,6 +50,7 @@ def get_total_stats(username):
     SELECT
         users.user_id,
         COUNT(run_id) AS total_runs,
+        COUNT(case finished when 1 then 1 else null end) AS total_completed_runs,
```

Comment:
dang is this really the sql case syntax?

Reply:
I couldnt figure out anything else for this, but theres probably a more elegant way that I'm not aware of. This works fine tho

Reply:
Here's where I got it: https://stackoverflow.com/questions/1400078/is-it-possible-to-specify-condition-in-count